LEAF_Request_Portal/utils/LEAF_importStandardConfig.php

```diff
@@ -5,16 +5,17 @@
 
 $debug = false;
 
-$res = $db->query_kv('SELECT * FROM settings', 'setting', 'data');
-
 // For Jira Ticket:LEAF-2471/remove-all-http-redirects-from-code
 //$protocol = isset($_SERVER['HTTPS']) ? 'https://' : 'http://';
 $protocol = 'https://';
 $siteRootURL = $protocol . HTTP_HOST;
-$relativePath = trim(str_replace($siteRootURL, '', $res['national_linkedPrimary']));
+
+$relativePath = trim(str_replace($siteRootURL, '', $settings['national_linkedPrimary']));

```

Comment:
Side note: If $settings is being initialized by the autoloader, we need to change the syntax to make it more obvious. For example rename to LEAF_SETTINGS. This would change it to a constant, and also uses a common all-caps convention for global declarations.

Reply:
This global was created and added to this file. I created another Jira ticket to make it universal. $settings is used a lot throughout the code base.